### DISCUSSION TITLE
python call <empty> query storage, how to check?

### DISCUSSION THREAD

**Discussion by yytomives** [polkascan/py-substrate-interface#296] [Q&A] [answered]
python call <empty> query storage, how to check? As shown below:
![image](https://user-images.githubusercontent.com/90444376/163214505-2dc66368-0a21-45bb-ac0e-43dd0db14946.png)

        call = substrate.query(
            module='System',
            storage_function='Account',
            params=[""]
        )
The value of params is ""? or None? or Null? I've tried both.

**Comment by arjanz** [comment, marked as answer]
You can use the `query_map()` function for that. Link to the doc: https://polkascan.github.io/py-substrate-interface/#substrateinterface.SubstrateInterface.query_map
